.github/pull_request_template.md

```diff
@@ -0,0 +1,28 @@
+Fixes #ISSUE.
```

Comment:
```suggestion
Resolves #ISSUE(s).
<!--- "Resolves #ISSUE" will automatically close #ISSUE when the PR is merged. However, if this PR won't 100% address the issue, you can use one of these patterns instead to simply link to the issue and not auto-close it:

Partially addresses:
- ISSUE(s)

Addresses sub-task (n) in:
- ISSUE(s)
-->
```

Reply:
Even though everything after the first line break is white, and before it is great, I double checked in _Preview_, and it looks like the whole thing is indeed a comment.